python/tvm/relay/tensorrt.py

```diff
@@ -220,24 +220,30 @@ def register_tensorrt_annotations(trt_version, use_implicit_batch=True):
     # _register_external_op_helper("slice_like")
 
     def add_whitelist_fn(attrs, args):  # pylint: disable=unused-variable
+        static_add = True
```

Comment:
Instead of having separate code paths to handle static vs dynamic add, lets process the shape first to replace Any with -1:

`shapes= [[int(x) if not isinstance(x, tvm.tir.expr.Any) else -1 for x in args.checked_type.shape] for arg in args]`

Then in all further code we only need to look at `shapes`.